lessons/10_conditions/index.html

```diff
@@ -0,0 +1,512 @@
+<html>
+
+<head>
+	<title>Lesson 10</title>
+	<link rel="stylesheet" href="../../reveal.js/css/reveal.css">
+	<link rel="stylesheet" href="../../reveal.js/css/theme/moon.css">
+	<link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js">
+	<style>
+		b {
+			background-color: #ad0170;
+		}
+
+		.reveal pre {
+			margin: 0;
+		}
+
+		.monospace {
+			font-family: monospace !important;
+		}
+
+		.illustration {
+			border: 0px
+		}
+
+		.reveal li {
+			margin-bottom: 0.5em;
+		}
+
+		.reveal pre {
+			width: 100%;
+		}
+
+		.reveal pre code {
+			cursor: pointer;
+		}
+
+		.reveal li pre {
+			width: fit-content;
+		}
+
+		.logo {
+			position: fixed;
+			/* Sit on top of the page content */
+			z-index: 2;
+			/* Specify a stack order in case you're using a different order for other elements */
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+	<div class="reveal">
+		<div class="slides">
+			<section>
+				<h1>Lecture 10</h1>
+				<h3>Conditions</h3>
+			</section>
+			<section>
+				<h3>What we would cover?</h3>
+				<ul>
+					<li class="fragment">If else blocks</li>
+					<li class="fragment">String comparison</li>
+					<li class="fragment">Switch case blocks</li>
+					<li class="fragment">Number comparison</li>
+					<li class="fragment">Practice</li>
+				</ul>
+			</section>
+			<section>
+				<h3>Time for a super-hero challenge 🦸</h3>
+				<ul>
+					<li class="fragment">You have just joined the Avengers team</li>
+					<li class="fragment">Time to greet all your new team mates</li>
+					<li class="fragment"><pre><code data-trim>
+						let name = "Spider-Man";
+						alert("Hi " + name + "!");</code></pre>
+					</li>
+					<li class="fragment"><b>Challenge:</b> Do not reveal super-hero names when not on a mission</li>
+					<li class="fragment"><pre><code data-trim>
+						let realName = "Peter Parker"
+						let heroName = "Spider-Man";
+						alert("Hi " + ??? + "!");</code></pre>
+					</li>
+				</ul>
+			</section>
+			<section>
+				<h3><code>if</code> Statements</h3>
+				<ul>
+					<li class="fragment">So far we are all in class</li>
+					<li class="fragment">But what <code>if</code> we go on a mission 🤪</li>
+					<li class="fragment">It runs code only under certain <b>conditions</b></li>
+					<li class="fragment">For conditions, you can e.g. compare two variables</li>
+					<li class="fragment">For equality we use <code>===</code> (yes, triple! 👀)</li>
```

Comment:
Would you explain why it's a tripple?

Reply:
I think, It's better to  explain == and  ===.

Reply:
Idk if coercion `==` is understandable for them yet. IMO they should learn the difference and not default to `===`, but you would need to understand javascript's type system

Reply:
I will mention it today, and maybe we can explain it in more details over some next lecture